### PR TITLE
Ensure reinitialize appSettings after standby mode

### PR DESCRIPTION
**Pull request by suwatch**
We need to reinitialize appSettings once switch from standby warmup to actual mode.  This change scopes to Azure only.

### CODE REVIEW

File: src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs [hunk 1]
```diff
@@ -138,6 +146,20 @@ private static void EnsureInitialize(WebHostSettings settings)
             }
         }
 
+        private static void ReinitializeAppSettings()
+        {
+            // only in azure environment
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("WEBSITE_SITE_NAME")))
```
Comment (davidebbo) [inline src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs]: You can use `Environment.IsAzureEnvironment` instead.

**Comment by davidebbo** [comment]
:shipit: just one small suggestion.

File: src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs (hunk 1 above)
Comment (suwatch) [inline src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs]: Um ..,  I couldn't find the routine here.  We have one in [kudu](https://github.com/projectkudu/kudu/blob/c0cbe58d9fab04bafd88300590e666f029d47942/Kudu.Core/Environment.cs#L317-L320) but not in webjob sdk.

Comment (davidebbo) [inline src/WebJobs.Script.WebHost/App_Start/WebHostResolver.cs]: My bad, forgot this wasn't Kudu! Might be worth adding a helper if there are other places in the code where we check for Azure.

**Pull request merged**